common/src/main/java/com/tc/config/schema/setup/LogSettingConfigItemListener.java

```diff
@@ -28,16 +28,10 @@
  */
 public class LogSettingConfigItemListener implements ConfigItemListener {
 
-  private final int processType;
-
-  public LogSettingConfigItemListener(int processType) {
-    this.processType = processType;
-  }
-
   @Override
   public void valueChanged(Object oldValue, Object newValue) {
     if (newValue != null) {
-      TCLogging.setLogDirectory((File) newValue, processType);
+      TCLogging.setLogDirectory((File) newValue, "terracotta-server");
```

Comment:
I'm thinking this class would be cleaner if you passed the log name in through the constructor, in the same way that the old code took the integer.